Client/src/test/java/org/gluu/oxauth/ws/rs/RegistrationWithSoftwareStatement.java

```diff
@@ -107,7 +107,7 @@ public void requestClientAssociate1(final String redirectUris, final String sect
         assertNotNull(response.getClaims().get(FRONT_CHANNEL_LOGOUT_SESSION_REQUIRED.toString()));
         assertTrue(Boolean.parseBoolean(response.getClaims().get(FRONT_CHANNEL_LOGOUT_SESSION_REQUIRED.toString())));
         assertNotNull(response.getClaims().get(FRONT_CHANNEL_LOGOUT_URI.toString()));
-        assertEquals(response.getClaims().get(FRONT_CHANNEL_LOGOUT_URI.toString()), logoutUri);
+        assertTrue(response.getClaims().get(FRONT_CHANNEL_LOGOUT_URI.toString()).equals(logoutUri));
```

Comment:
Minor but assertEquals seems to fit better (just leave unchanged).

Reply:
Done